dynamic-geoassets-api/internal/app/about.go

```diff
@@ -0,0 +1,17 @@
+package app
+
+import (
+	"net/url"
+
+	"github.com/metno/S-ENDA-documentation/dynamic-geoassets-api/pkg/metaservice"
+)
+
+func about() *metaservice.About {
+	return &metaservice.About{
+		Name:           "Dynamic geo assets API",
+		Description:    "The purpose of this service is to list, create and handle datasets and services for geodynamic assets.",
+		Responsible:    "S-ENDA sprint team <s-enda-sprint@met.no>",
+		Documentation:  &url.URL{Path: "/"},
+		TermsOfService: &url.URL{Path: "/docs/termsofservice"},
+	}
+}
```

Comment:
Do these come from the MET template?

Reply:
No, its something I have been testing out. E.g, each service exposes metadata about itself on a well known endpoint  `<basePath>/about`. 

It could also be used by this system to harvest metadata from services, after the base path has been registered.